plugin.yml

```diff
@@ -10,6 +10,12 @@ configuration:
   properties:
     server:
       type: string
+    auth-method:
+      type: string
```

Comment:
why not ensure that, if provided, the value is one of the supported ones (empty or `approle`)?

Reply:
Good suggestion! done ✅